tests/utils/test_add_new_model_like.py

```diff
@@ -85,6 +85,7 @@
 @require_torch
 @require_tf
 @require_flax
+@unittest.skip("The file `add_new_model_like.py` requires an installation with `-e` flag.")
```

Comment:
This tests fails in the new job `non_modeling_job` as it has all 3 libs. installed. Before this PR, this test is never run by CI as we don't have a job targets this tests with all necessary libs. in it.

The problem is that the file `src/transformers/commands/add_new_model_like.py` has 
```python
TRANSFORMERS_PATH = Path(__file__).parent.parent
REPO_PATH = TRANSFORMERS_PATH.parent.parent
```
which (seems to me) requires an `-e` installation, but on CircleCI, we have
```bash
TRANSFORMERS_PATH  = /home/circleci/.pyenv/versions/3.8.12/lib/python3.8/site-packages/transformers
```
which causes problem.

Reply:
Would like to have you opinion about how to address this.

Reply:
I don't think we can do an editable install on circleCI sadly, so no idea how to test this one.